core/storage/suggestion/gae_models.py

```diff
@@ -2360,6 +2362,8 @@ def fetch_page(
                     cls.language_code == language_code
                 )).order(sort)
 
+        total_records = sort_query.count()
```

Comment:
If results exist, should we be adding the offset to this number?

Conceptually, what should the value here be if the given offset is too large? Probably None to indicate "no data"? (If so please explain this in the docstring.)

There should be tests for all these cases, too.